ci/patch_src_in_place

```diff
@@ -17,9 +23,14 @@ if [ -x $CP ]
 then
     for FILE in $(git ls-tree --full-tree --name-only HEAD -r src | grep -v -e src/control/vendor -e .go\$ -e pb-c -e debug_setup.h)
     do
-        $CP --quiet --no-tree -file --show-types --no-summary --ignore "$def_ignore" --strict "$FILE" --fix-inplace
+	ignore="$def_ignore"
+	grep -lq CRT_RPC_DECLARE "$FILE"
+	if [ $? -eq 0 ]
+	then
+	    ignore+=",SPACING_CAST"
+	fi
+        $CP --quiet --no-tree -file --show-types --no-summary --ignore "$ignore" --strict "$FILE" --fix-inplace
```

Comment:
Is `-file` an option?  If so, why does it have only one `-` when all other options have two?

Reply:
Good point, seems to work but I'll change this.